deployment.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Deployment
+layout: recommendation
+expires: 2017-12-01
+---
+
```

Comment:
Describe the need: "so that we can be confident of what code is running, and be able to run it again".